src/platforms/python/guides/gcp-functions/index.mdx

```diff
@@ -60,7 +62,8 @@ The timeout warning is sent only if the timeout in the GCP Function configuratio
 
 With the GCP Functions integration enabled, the Python SDK will:
 
-- Automatically report all exceptions from your google cloud functions
+- Automatically report all events from your Lambda Functions
+- You can control transactions sample rate using `traces_sample_rate`. You can learn more about transactions in <PlatformLink to="/configuration/filtering/">Distributed Tracing</PlatformLink>
```

Comment:
```suggestion
- Automatically report all events from your Lambda Functions. You can [modify the transaction sample rate] <PlatformLink to="/configuration/filtering/">Distributed Tracing</PlatformLink> using `traces_sample_rate`.
```

Reply:
The rephrasing of point 2 looks good but the first and second bulletpoint are not related to each other in any way.